scripts/checker.py

```diff
@@ -75,27 +82,44 @@ def do_check(row):
     context["cs_path_info"] = row["path"]
     context["cs_username"] = row["username"]
     context["cs_user_info"] = {"username": row["username"]}
+    context['is_running_checker'] = True			# may be used by preload.py
     context["cs_user_info"] = auth.get_user_information(context)
     context["cs_now"] = datetime.fromtimestamp(row["time"])
+
+    have_lti = ('cs_lti_config' in context) and ('lti_data' in row)
```

Comment:
It's possible that we may need something like this in `handle_submission` as well, in the part for handling the `'legacy'` grading mode (which skips the asynchronous checker).  Maybe this could be abstracted into a function in `lti.py`, and then we could just call that with the appropriate arguments from both places?